dev/sparktestsupport/toposort.py

```diff
@@ -0,0 +1,85 @@
+#######################################################################
+# Implements a topological sort algorithm.
```

Comment:
@srowen, I didn't want to write my own sort so I just copied this one from the [toposort](https://bitbucket.org/ericvsmith/toposort) library. Do I need to update the `NOTICE` file in this case?